compiler-plugin/src/main/java/io/ballerina/stdlib/websocket/plugin/Utils.java

```diff
@@ -338,12 +338,10 @@ private static void repoteDiagnostics(String functionName, FunctionDefinitionNod
     }
 
     private static boolean validateReturnType(TypeSymbol symbol) {
-        return !(symbol.typeKind() == TypeDescKind.ERROR) && !(symbol.typeKind() == TypeDescKind.TYPE_REFERENCE)
-                && (symbol.signature().contains(ERROR)) && !(symbol.typeKind() == TypeDescKind.NIL)
-                && !(symbol.typeKind() == TypeDescKind.STRING) && !(symbol.typeKind() == TypeDescKind.ARRAY)
-                && !(symbol.typeKind() == TypeDescKind.INT) && !(symbol.typeKind() == TypeDescKind.BOOLEAN)
-                && !(symbol.typeKind() == TypeDescKind.DECIMAL) && !(symbol.typeKind() == TypeDescKind.JSON)
-                && !(symbol.typeKind() == TypeDescKind.XML) && !(symbol.typeKind() == TypeDescKind.FLOAT);
+        TypeDescKind[] validReturnTypes = {TypeDescKind.ERROR, TypeDescKind.TYPE_REFERENCE, TypeDescKind.STREAM,
+                TypeDescKind.NIL, TypeDescKind.STRING, TypeDescKind.ARRAY, TypeDescKind.INT, TypeDescKind.BOOLEAN,
+                TypeDescKind.DECIMAL, TypeDescKind.JSON, TypeDescKind.XML, TypeDescKind.FLOAT};
+        return !Arrays.asList(validReturnTypes).contains(symbol.typeKind());
```

Comment:
This function name is not ideal. Since it is returning a boolean value, maybe we can rename this as `isValidReturnType` or `isInvalidReturnType`?

```suggestion
        List<TypeDescKind> validReturnTypes = Arrays.asList(TypeDescKind.ERROR, TypeDescKind.TYPE_REFERENCE, TypeDescKind.STREAM,
                TypeDescKind.NIL, TypeDescKind.STRING, TypeDescKind.ARRAY, TypeDescKind.INT, TypeDescKind.BOOLEAN,
                TypeDescKind.DECIMAL, TypeDescKind.JSON, TypeDescKind.XML, TypeDescKind.FLOAT);
        return !validReturnTypes.contains(symbol.typeKind());
```

Reply:
refactored with [f234e77](https://github.com/ballerina-platform/module-ballerina-websocket/pull/1121/commits/f234e7728c34c2ba7e04ad63351b05d260fde3da)